common-lib/src/main/scala/com/gu/mediaservice/model/leases/LeasesByMedia.scala

```diff
@@ -0,0 +1,46 @@
+package com.gu.mediaservice.model
+
+import play.api.libs.json._
+import org.joda.time.DateTime
+import JodaWrites._
+
+case class LeasesByMedia(
+  leases: List[MediaLease],
+  lastModified: Option[DateTime]
+)
+
+object LeasesByMedia {
+  implicit val reader : Reads[LeasesByMedia] = (__ \ "leases").read[List[MediaLease]].map(LeasesByMedia.build)
+
+  implicit val writer = new Writes[LeasesByMedia] {
+    def writes(leaseByMedia: LeasesByMedia) = {
+      LeasesByMedia.toJson(
+        Json.toJson(leaseByMedia.leases),
+        Json.toJson(leaseByMedia.lastModified.map(lm => Json.toJson(lm)))
+      )
+    }
+  }
+
+  def build(leases: List[MediaLease]): LeasesByMedia = {
+    def sortLease(a: MediaLease, b: MediaLease) =
+      a.createdAt.isAfter(b.createdAt)
+
+    val sortedLeases = leases
+      .sortWith(sortLease)
```

Comment:
Minor: I'm not sure from reading this which order it is sorting in (ascending or descending). I think it's descending?

Is it clearer with sortBy?

```scala
val lastModified = leases.sortBy(_.createdAt).reverse.headOption
```